src/main/java/game/GameEngine.java

```diff
@@ -109,14 +109,15 @@ private static void startGameLoop(Map p_map) {
     /**
      * <p>The method assign army's to each player</p>
      * @param p_map map for the game
+     * @author Naveen
      */
-    private static void assignReinforcements(Map p_map) {
+    public static void assignReinforcements(Map p_map) {
 
-        final int l_REINFORCEMENTS_PER_PLAYER = 5; // Number of reinforcements per player
+        final int l_REINFORCEMENTS_PER_PLAYER = 5; // Number of armies per player
 
         for (Player l_player : p_map.getD_players()) {
```

Comment:
This is not sufficient. As we discussed in the call, you also have to check if a player owns a whole continent and give him the bonus.